File: crud-python/Ex02_Connect_to_Cassandra.py
#!/usr/bin/env python3
from db_connection import Connection

print('========================================')
print('Start exercise')
try:
	connection = Connection()
	output = connection.session.execute("SELECT * FROM system.local")
	for row in output:
	    print("Your are now connected to cluster '{}'".format(row.cluster_name))
except Exception as e:
    print(e)
    print('Failure')
else:
    print('Success')
finally:
	print('Closing connection (up to 10s)')
	connection.close()
print('========================================')

File: crud-nodejs/Ex02_Connect_to_Cassandra.js
const connection = require('./db_connection')
const Uuid       = require('cassandra-driver').types.Uuid;

console.log("========================================")
console.log("Start exercise")

connection.client
.execute('SELECT * FROM system.local')
.then(function(result) {
	result.rows.forEach(row => {
        console.log("Your are now connected to cluster '%s'", row.cluster_name)
    })
    connection.client.shutdown()
    console.log("SUCCESS")
})
.catch(function(error){
    console.log(error.message)
    connection.client.shutdown()
});

console.log("========================================")